skonfig/arguments.py

```diff
@@ -41,8 +41,9 @@ def get():
     )
     parser.add_argument("host", nargs="?", help="host to configure")
     arguments = parser.parse_args()
-    if arguments.verbose >= 2:
-        logging.basicConfig(level=logging.DEBUG)
+    if arguments.verbose > 3:
+        arguments.verbose = 3
+    logging.basicConfig(level=[20, 15, 10, 5][arguments.verbose])
```

Comment:
This is hackish, because `cdist` defines its own additional levels - 15 = VERBOSE and 5 = TRACE. But same time it feels elegant hack.

Reply:
The only thing I don't like about it are the hard coded logging levels, because they could get out of sync with `cdist.log`.

Maybe we could replace the level list with the actual levels instead?

Not exactly elegant, but something like this should work: 
```python
levels = list(reversed(sorted(
    l for (l, n) in getattr(logging, "_levelToName", getattr(logging, "_levelNames", {})).items()
    if isinstance(l, int) and n != "NOTSET" and l <= logging.getLevelName("INFO"))))
level = levels[min(arguments.verbose, len(levels)-1)]
logging.basicConfig(level=level)
`````

Reply:
I think this part should be in separate PR.